UI.PageModels/Pages/Admin/Booking/ParticipantsPage.cs

```diff
@@ -154,4 +158,17 @@ public VideoAccessPointsPage GoToVideoAccessPointsPage()
         ClickElement(_nextButton);
         return new VideoAccessPointsPage(Driver, DefaultWaitTime);
     }
+    
+    public SummaryPage GoToSummaryPage()
+    {
+        ClickElement(_nextButton);
+        return new SummaryPage(Driver, DefaultWaitTime);
+    }
+    
+    public void CompleteBookAVideoHearing(string displayName, string advocate)
```

Comment:
This has nothing to do with completing a booking. This is the Participant's page, where only managing the participant list should be concerned

Reply:
we can move it from this Participant page class but you have place Video access element in that section